common/src/main/java/fr/openent/presences/common/helper/PersonHelper.java

```diff
@@ -60,7 +60,7 @@ public List<User> getUserListFromJsonArray(JsonArray userArray) {
      */
     public void getStudentsInfo(String structureId, List<String> studentIds, List<String> restrictedClasses, Handler<Either<String, JsonArray>> handler) {
         String query = "MATCH (s:Structure {id: {structureId} })<--()--" +
-                "(u:User {profiles:['Student']})-[:IN]->(:ProfileGroup)-[:DEPENDS]->(c:Class) WHERE u.id IN {idStudents} ";
+                "(u:User {profiles:['Student']})-[:IN]->(:ProfileGroup)-[:DEPENDS]->(c:Class)-[:BELONGS]->(s) WHERE u.id IN {idStudents} ";
```

Comment:
MATCH (u:User {profiles:['Student']})-[:IN]->(:ProfileGroup)-[:DEPENDS]->(c:Class)-[:BELONGS]->(s:Structure {id: {structureId} })

Reply:
Yes justement, c'est ce qu'il m'a perturbé.
J'approve BTW